### DISCUSSION TITLE
Support Collapsible Entries in Markdown

### DISCUSSION THREAD

**Discussion by proskehy** [GitJournal/GitJournal#825] [Feature Requests]
Using a collapsible markdown as showcased [here](https://gist.github.com/joyrexus/16041f2426450e73f5df9391f7f7ae5f) does not show as collapsible in the iOS app (I have not tested in Android). The device this was tested on is iPhone SE with iOS 13.3.1.

A simple snippet like this:
<details><summary>Collapse title</summary>

### Collapse body

  - item1
  - item2

</details>

Code:

```
<details><summary>Collapse title</summary>

### Collapse body

  - item1
  - item2

</details>
```

Shows up like this and is not clickable:
<img width="320" alt="screenshot" src="https://user-images.githubusercontent.com/24292015/78929891-763a1f80-7aa3-11ea-8597-d95ebaeefc0d.png">

Is this a bug or is that functionality not supported? 

---
> Please vote on this issue with a 👍reaction. It's one of the primary mechanisms used to prioritize issues.

**Comment by vHanda** [comment]
Hi. This functionality still isn't supported.

Anything that involves HTML tags is currently not supported. My main aim right now is just to support the entire base and extended syntax of markdown - https://www.markdownguide.org/extended-syntax/

I'm leaving this bug open, but I have no immediate plans to tackle it. Unless lots of others users also need this functionality.
